assets/javascripts/tests.js

```diff
@@ -554,3 +556,9 @@ function showJobDependency(deps) {
   }
   return result;
 }
+
+$(document).on('click', '.copy-jobid', function (event) {
+  event.preventDefault();
+  var id = $(this).attr('data-jobid');
+  navigator.clipboard.writeText(id);
```

Comment:
Use of `var` should be avoided in new code, as well as use of jQuery when it is useless anyways.

```suggestion
  navigator.clipboard.writeText(this.dataset.jobid);
```